test/functional/api/window_spec.lua

```diff
@@ -1292,6 +1723,154 @@ describe('API/win', function()
       eq(title, cfg.title)
       eq(footer, cfg.footer)
     end)
+
+    it('includes split for normal windows', function()
+      local win = api.nvim_open_win(0, true, {
+        vertical = true,
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'left')
+      api.nvim_win_set_config(win, {
+        vertical = false,
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'above')
+      api.nvim_win_set_config(win, {
+        split = 'below',
+        win = -1,
+      })
+      eq(api.nvim_win_get_config(win).split, 'below')
+    end)
+
+    it('includes split when splitting with ex commands', function()
+      local win = api.nvim_get_current_win()
+      eq(api.nvim_win_get_config(win).split, 'left')
+
+      command('vsplit')
+      local win2 = api.nvim_get_current_win()
+
+      -- initial window now be marked as right split
+      -- since it was split with a vertical split
+      -- and 'splitright' is false by default
+      eq(api.nvim_win_get_config(win).split, 'right')
+      eq(api.nvim_win_get_config(win2).split, 'left')
+
+      api.nvim_set_option_value('splitbelow', true, {
+        scope = 'global',
+      })
+      api.nvim_win_close(win, true)
+      command('split')
+      local win3 = api.nvim_get_current_win()
+      eq(api.nvim_win_get_config(win3).split, 'below')
+    end)
+
+    it("includes the correct 'split' option in complex layouts", function()
+      local initial_win = api.nvim_get_current_win()
+      local win = api.nvim_open_win(0, false, {
+        split = 'right',
+        win = -1,
+      })
+
+      local win2 = api.nvim_open_win(0, false, {
+        split = 'below',
+        win = win,
+      })
+
+      api.nvim_win_set_config(win2, {
+        width = 50,
+      })
+
+      api.nvim_win_set_config(win, {
+        split = 'left',
+        win = -1,
+      })
+
+      local win3 = api.nvim_open_win(0, false, {
+        split = 'above',
+        win = -1,
+      })
+      local float = api.nvim_open_win(0, false, {
+        relative = 'editor',
+        width = 40,
+        height = 20,
+        col = 20,
+        row = 10,
+      })
+      api.nvim_win_set_config(float, {
+        split = 'right',
+        win = -1,
+      })
+
+      local layout = api.nvim_call_function('winlayout', {})
```

Comment:
```suggestion
      local layout = fn.winlayout()
```

Reply:
Is there a practical difference here or just appearance? (I have nothing against this change, just wondering so I can know for the future)

Reply:
Just appearance.